lib/sinon/proxy.js

```diff
@@ -253,7 +254,7 @@ function createProxy(func, originalFunc) {
     return proxy;
 }
 
-function wrapFunction(func, originalFunc) {
+function wrapFunction(func, originalFunc, ctx = globalContext) {
```

Comment:
Why are all of these context objects optional, with a default global? Even the default Sinon instance is a sandbox in itself.